x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1330,4 +1330,69 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'mac.advanced.kernel.fileaccess',
+    first_supported_version: '8.11',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.kernel.fileaccess',
+      {
+        defaultMessage: 'Sends file access events to the stack. Default: false',
```

Comment:
Based on your answer below, I think this text is clearer. Its the type of text used for similar options (like `mac.advanced.kernel.process`

```suggestion
        defaultMessage: 'A value of 'false' overrides other config settings that would enable kernel fileaccess events. Default: true.`,
```